execute.c

```diff
@@ -536,17 +555,30 @@ jv jq_next(jq_state *jq) {
       stack_save(jq, pc - 1, stack_get_pos(jq));
 
       stack_push(jq, jv_number(jq->subexp_nest));
+      stack_push(jq, jv_copy(jq->value_at_path));
       stack_push(jq, v);
```

Comment:
No it doesn't, but I will move jv_copy from line 563 to here. I guess the reason I had it below was I conceptualized v as "a value on the stack," even though we temporarily pop it off to insert things below it. So the stack is holding a reference to v. On line 563, also putting a reference to v into the jq_state requires a jv_copy.